reminder/module.py

```diff
@@ -252,6 +261,15 @@ async def remind(
             )
             return
 
+        if date < datetime.now():
+            await ctx.reply(
+                _(
+                    ctx,
+                    "Time already passed.",
```

Comment:
Actually, this message already exists in 'reminder reschedule' as

> Reschedule time must be in furuter.

Please, use

> Time must be in future.

for all of the messages.

Reply:
OK, done.